connect/cli/plugins/commerce/utils.py

```diff
@@ -0,0 +1,340 @@
+        raise ClickException(f'Error obtaining file {file_id} -> {file_destination}')
+    with open(file_destination, 'wb') as f:
+        f.write(response)
+    console.secho(
+        f'File {file_destination} downloaded properly.',
+        fg='green',
+    )
+
+
+def fill_and_download_attachments(ws, attachments, client, base_path, progress):
+    attachments = list(attachments)
+    ws.column_dimensions['A'].width = 30
+    ws.column_dimensions['B'].width = 30
+    _fill_headers(
+        ws,
+        ('ID', 'Name'),
+    )
+    task = progress.add_task('Filling and downloading attachments', total=len(attachments))
+    for n, attachment in enumerate(attachments, 2):
+        ws.cell(n, 1, value=attachment['id'])
+        ws.cell(n, 2, value=attachment['name'])
+        _download_file(
+            client=client,
+            folder_type=attachment['folder']['type'],
+            folder_name=attachment['folder']['name'],
+            file_id=attachment['id'],
+            file_destination=os.path.join(base_path, attachment['name']),
+        )
+        progress.update(task, advance=1)
+
+
+def export_stream(
+    client,
+    stream_id,
+    active_account_id,
+    progress,
+    output_file,
+    output_path=None,
+):
+    collection = guess_if_billing_or_pricing_stream(client, stream_id)
+    if not collection:
+        console.secho(
+            f'Stream {stream_id} not found for the current account {active_account_id}.',
+            fg='red',
+        )
+        return
+
+    response = (
+        client.ns(collection)
+        .streams.filter(id=stream_id)
+        .select(
+            'context',
+            'samples',
+            'sources',
+        )
+        .first()
+    )
+
+    wb = Workbook()
+
+    output_file = validate_output_options(output_path, output_file, default_dir_name=stream_id)
+    attachments_path = os.path.join(os.path.dirname(output_file), 'attachments')
+    if not os.path.exists(attachments_path):
+        os.mkdir(attachments_path)
+
+    sample_input_path = os.path.join(os.path.dirname(output_file), 'sample', 'input')
+    if not os.path.exists(sample_input_path):
+        os.makedirs(sample_input_path)
+
+    stream_type = 'Computed' if 'sources' in response and response['sources'] else 'Simple'
+    input = response['samples'].get('input', {})
+    input_file_name = input['name'].split('/')[-1]
+    fill_general_information(
+        ws=wb.active,
+        data={
+            'Stream ID': stream_id,
+            'Stream Name': response['name'],
+            'Stream Description': response['description'],
+            'Stream Type': stream_type,
+            'Stream Category': 'Inbound'
+            if response['owner']['id'] == active_account_id
+            else 'Outbound',
+            'Computed Stream Source ID': response['sources'][0]['id']
+            if stream_type == 'Computed'
+            else '',
+            'Computed Stream Source Name': response['sources'][0]['type']
```

Comment:
Not `type` probably `name` should be taken here.

Reply:
Agree. So I will add also the name. We will export id, name and type.